### PR TITLE
Add BaristaSeq tilefetcher

### PR DESCRIPTION
**Pull request by ambrosejcarr**
For BaristaSeq, We previously wrote this TileFetcher, uploaded data to `s3`, and added BaristaSeq to `starfish.data`, however the TileFetcher code was not uploaded. 

This PR adds the TileFetcher code.

### CODE REVIEW

File: examples/format_baristaseq.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        return BaristaSeqTile(file_path)
+
+
+class BaristaSeqNucleiTileFetcher(TileFetcher):
+    def __init__(self, input_dir, aux_type) -> None:
+        self.input_dir = input_dir
+
+    def get_tile(self, fov: int, hyb: int, ch: int, z: int) -> FetchedTile:
+        subdir = 'nissl'
+        filename = f'T00001C05Z{z+1:03}.tif'
```
Comment (shanaxel42) [inline examples/format_baristaseq.py]: will these ever change?

File: examples/format_baristaseq.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            Coordinates.Z: (0.0, 0.0001),
+        }
+
+    @property
+    def format(self) -> ImageFormat:
+        return ImageFormat.TIFF
+
+    @property
+    def tile_data(self) -> Callable[[], np.ndarray]:
+        def future():
```
[the remark this answers is not in the document]
Comment (ambrosejcarr) [reply examples/format_baristaseq.py]: 🤦‍♂️

[the remark this answers is not in the document]
Comment (ambrosejcarr) [reply examples/format_baristaseq.py]: I owe you a beer.

[the remark this answers is not in the document]
Comment (ambrosejcarr) [reply examples/format_baristaseq.py]: This might be the most embarrassing code I've ever written.

File: examples/format_baristaseq.py (hunk 1 above)
Comment (ambrosejcarr) [reply examples/format_baristaseq.py]: The file names? No, not for this dataset. But generally, all TileFetcher code is one-off. It's worthless if the user structures their directory differently or changes file naming procedures for _future datasets_.

File: examples/format_baristaseq.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    def __init__(self, file_path):
+        self.file_path = file_path
+
+    @property
+    def shape(self) -> Tuple[int, ...]:
+        return DEFAULT_TILE_SHAPE
+
+    @property
+    def coordinates(self) -> Mapping[Union[str, Coordinates], Union[Number, Tuple[Number, Number]]]:
+        # these are dummy coordinates
```
[the remark this answers is not in the document]
Comment (ambrosejcarr) [reply examples/format_baristaseq.py]: No they do not. I don't think it's worth my time or our collaborator's for single FOV datasets. :-(

File: examples/format_baristaseq.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    output_dir : str
+        Output directory containing formatted data in SpaceTx format. Example output data can be
+        downloaded from
+        https://d2nhj9g34unfro.cloudfront.net/browse/formatted/20181028/ \
+        BaristaSeq/cropped_formatted/experiment.json"
+    """
+
+    def add_codebook(experiment_json_doc):
+        # this just needs to be copied over, but here we make the link
+        experiment_json_doc['codebook'] = "codebook.json"
```
[the remark this answers is not in the document]
Comment (ambrosejcarr) [reply examples/format_baristaseq.py]: Also went back and fixed `SeqFISH`. Thanks for the comment.

**Pull request merged**